doc/Makefile.in

```diff
@@ -116,8 +115,8 @@ XSLTPROC_CHUNKED_HTML_OPTS = \
 	--stringparam img.src.path ../ \
 	--stringparam html.stylesheet ../style.css
 
-HTML_DOCBOOK_XSL=$(XSLBASE)/xhtml5/docbook.xsl
-CHUNK_HTML_DOCBOOK_XSL=$(XSLBASE)/xhtml5/chunk.xsl
+HTML_DOCBOOK_XSL=xsl/docbook-html-utf8.xsl
```

Comment:
You'll probably need $(srcdir) or $(abs_srcdir) here, check both `make html` and `make html-localized` to be sure, from a build tree outside of source tree :)